Sprint 04

Azonaz opened the pull request:
Выполненное задание по 4 спринту на ревью

Azonaz committed:
Завершение задачи по спринту 4

Azonaz committed:
Реформат отступов.
Избавиться от предупреждения в сториборде

protos17 commented on MovieQuiz/Presentation/MovieQuizViewController.swift:
```diff
@@ -1,72 +1,171 @@
+            correctAnswer: false),
+        QuizQuestion(
+            image: "Vivarium",
+            text: "Рейтинг этого фильма больше чем 6?",
```
Повторяющуюся строку рекоменду вынести в отдельную константу

Azonaz committed:
Внесены исправления по замечаниям ревьюра:
- изменен шрифт на кнопках
- убран дублирующий текст
- изменен цвет текста в статус баре на светлый (как в макете)

Azonaz committed:
Рефакторинг: убран повторяющийся параметр
